client/my-sites/plans/jetpack-plans/plans-header.tsx

```diff
@@ -18,15 +19,17 @@ import getSiteProducts from 'calypso/state/sites/selectors/get-site-products';
 import { PLAN_JETPACK_FREE, JETPACK_PRODUCTS_LIST } from '@automattic/calypso-products';
 import IntroPricingBanner from 'calypso/components/jetpack/intro-pricing-banner';
 
-const StandardPlansHeader = () => (
+const StandardPlansHeader = ( { context }: { context: PageJS.Context } ) => (
 	<>
 		<FormattedHeader headerText={ translate( 'Plans' ) } align="left" brandFont />
 		<PlansNavigation path={ '/plans' } />
-		<h2 className="jetpack-plans__pricing-header">
-			{ preventWidows(
-				translate( 'Security, performance, and marketing tools made for WordPress' )
-			) }
-		</h2>
+		{ ! getPlanRecommendationFromContext( context ) && (
+			<h2 className="jetpack-plans__pricing-header">
+				{ preventWidows(
+					translate( 'Security, performance, and marketing tools made for WordPress' )
+				) }
+			</h2>
+		) }
```

Comment:
The purpose of `getPlanRecommendationFromContext` was hard to understand. Maybe we could rename it to something like `shouldShowPlansRecommendation` or something like that, and compute that value on the parent component. I don't see a strong reason to pass the whole context object when we can accomplish the same just by passing a boolean.

Reply:
Fair point. I'll update that.